index.html

```diff
@@ -103,6 +103,10 @@
         Model
       </h2>
       <p>
+        This section concepts used in this specification, and how those
```

Comment:
This section concepts used in this specification, and how those

Missing a word there. “This section **defines** concepts used in this specification” I guess